documentation/docs/steps/transportRequestUploadCTS.md

```diff
@@ -0,0 +1,173 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+* You have installed the SAP component SAP_UI 7.53 or higher on your ABAP system.
+* You have enabled the OData Service to Load Data to the [SAPUI5 ABAP Repository](https://sapui5.hana.ondemand.com/#/topic/a883327a82ef4cc792f3c1e7b7a48de8.html)
+* You have the [S_DEVELOP authorization](https://sapui5.hana.ondemand.com/#/topic/a883327a82ef4cc792f3c1e7b7a48de8.html) for operations on a SAPUI5 ABAP repository.
+* You have created a transport request on the ABAP system, which is the target container of the SAPUI5 application to upload.
+
+## Upload Client Setup
```

Comment:
Setting Up an Upload Client* (not sure about this, yet it is recommended to use the Gerund/-ing form in "how-to" titles)